components/Registry.tsx

```diff
@@ -519,9 +519,14 @@ function DirectoryListing(props: {
                           {entry.name}
                         </td>
                         <td className="px-4 py-1 whitespace-no-wrap text-sm leading-5 text-gray-500 text-right">
-                          {entry.size !== undefined
-                            ? bytesToSize(entry.size)
-                            : "N/A"}
+                          {
+                            /** TODO(kennanseno): Directory should pull total size inside it.  */
+                            entry.type === "dir"
+                              ? ""
+                              : entry.size !== undefined
+                              ? bytesToSize(entry.size)
+                              : "N/A"
```

Comment:
How about just blank instead of "N/A" ?

Reply:
empty string could just work to be honest but it's probably better to differentiate between a valid directory and a file for that some reason we can't resolve the size.

Reply:
> a file that for some reason we can't resolve the size

when does that happen?

Reply:
I'm just assuming based on the existing undefined check to entry.size.

Not a big deal tbh and happy to leave it to empty string either way.